composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,67 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
+
+client = bigquery.Client()
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
+dataset_id = f"{client.project}.sample_dataset_{dataset_UUID}"
+
+
+@pytest.fixture(scope="module")
+def dataset():
+
+    dataset = bigquery.Dataset(dataset_id)
+
+    try:
+        dataset = client.get_dataset(dataset_id)
+    except google.cloud.exceptions.NotFound:
+        dataset = client.create_dataset(dataset)
+
+    yield dataset
+
+    client.delete_dataset(dataset_id, delete_contents=True, not_found_ok=True)
+
+
+@pytest.fixture(scope="module")
+def table():
+
+    table_id = f"{dataset_id}.average_weather"
+
+    schema = [
+        bigquery.SchemaField("location", "GEOGRAPHY", mode="REQUIRED"),
+        bigquery.SchemaField("average_temperature", "INTEGER", mode="REQUIRED"),
+        bigquery.SchemaField("month", "STRING", mode="REQUIRED"),
+        bigquery.SchemaField("inches_of_rain", "NUMERIC", mode="NULLABLE"),
+        bigquery.SchemaField("is_current", "BOOLEAN", mode="NULLABLE"),
+        bigquery.SchemaField("latest_measurement", "DATE", mode="NULLABLE"),
+    ]
+
+    table = bigquery.Table(table_id, schema=schema)
+
+    try:
```

Comment:
Nit - I think that instead of catching for a NotFound exception, you can pass "exists_ok=True" as part of the [create command](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client.create_table)